internal/utils/utils.go

```diff
@@ -167,6 +167,11 @@ func GetInt64(value int64) *int64 {
 	return &i
 }
 
+func GetBool(value bool) *bool {
+	b := value
+	return &b
+}
+
```

Comment:
I suggest to stop adding custom helper as this one and start leveraging `"k8s.io/utils/pointer"` that supports `pointer.BoolPtr(true)` and others.